sinch/core/clients/sinch_client_configuration.py

```diff
@@ -19,14 +19,19 @@ def __init__(
         logger=None,
         logger_name: str = None,
         disable_https=False,
-        connection_timeout=10
+        connection_timeout=10,
+        verification_key: str = None,
+        verification_secret: str = None
```

Comment:
because of key and secret can be related to Voice too, can we imagine having them renamed to "application_key / application_secret"
(see current opened ADR comment and Java SDK PR)

Reply:
Cool idea 👍

Reply:
fixed